src/scripts/migrate-dep-order.ts

```diff
@@ -32,7 +32,7 @@ const entities = sortedDeps.map(d => `${__dirname}/../modules/${moduleVersion}/$
 
   for (const dep of sortedDeps) {
     console.log(`Adding ${dep.name}...`);
-    const migrationClass = getModMigration(dep.name, dep.version);
+    const migrationClass = getModMigration(dep.name, moduleVersion);
```

Comment:
Does this change do anything?

Reply:
Yes, after the update that removes the version from the `module.json` the generation of migrations was broken.  The `dep.version` was `undefined` here. This change fixes it.

Reply:
Ohh. Got it.